### PR TITLE
en: update dm-worker config doc for 2.0

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

update dm-worker-config-file.md for v2.0

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/25
- Other reference link(s):

### CODE REVIEW

**Comment by GMHDBJD** [comment]
The en/relay-log.md and zh/relay-log.md are inconsistent.  I think you can translate it or just remove the dead link because relay-log is no so perfect in 2.0 now.

**Comment by toutdesuite** [comment]
@GMHDBJD Thanks for your suggestion. The dead link is removed. PTAL~

File: en/dm-worker-configuration-file.md [hunk 1]
```diff
@@ -6,59 +6,35 @@ category: reference
 
 # DM-worker Configuration File
 
-This document introduces the basic configuration of DM worker, which provisions DM-worker's deployment in most scenarios. Refer to [DM-worker Advanced Configuration File](dm-worker-configuration-file-full.md) to see more parameters in detail.
+This document introduces the configuration of DM worker, including a configuration file template and instructions about what the configuration parameters mean.
```
Comment (TomShawn) [inline en/dm-worker-configuration-file.md:9]:
```suggestion
This document introduces the configuration of DM worker, including a configuration file template and a description of each configuration parameter in this file.
```

File: en/dm-worker-configuration-file.md [hunk 2]
```diff
@@ -6,59 +6,35 @@ category: reference
...
-The `[from]` section contains parameters that affect the connection to the upstream database.
-
 | Parameter        | Description                           |
 | :------------ | :--------------------------------------- |
-| `host` | The host name of the upstream database. |
-| `port` | The port number of the upstream database. |
-| `user` | The username used to connect to the database. |
-| `password` | The password used to connect to the database. Note: Use the password encrypted by dmctl.
\ No newline at end of file
+| `name` | Identifies a DM-worker. |
```
Comment (TomShawn) [inline en/dm-worker-configuration-file.md:35]:
```suggestion
| `name` | Marks a DM-worker name. |
```

File: en/dm-worker-configuration-file.md [hunk 3]
```diff
@@ -6,59 +6,35 @@ category: reference
...
+| `log-level` | Specifies a log level from `debug`, `info`, `warn`, `error`, and `fatal`. The default log level is `info`. |
```
Comment (TomShawn) [inline en/dm-worker-configuration-file.md:36]:
```suggestion
| `log-level` | Specifies the log level. The optional values are `debug`, `info`, `warn`, `error`, and `fatal`. The default log level is `info`. |
```

File: en/dm-worker-configuration-file.md [hunk 4]
```diff
@@ -6,59 +6,35 @@ category: reference
...
+| `log-file` | Specifies the log file directory. If not specified, the logs are printed onto the standard output. |
```
Comment (TomShawn) [inline en/dm-worker-configuration-file.md:37]:
```suggestion
| `log-file` | Specifies the log file directory. If this parameter is not specified, the logs are printed onto the standard output. |
```

File: en/dm-worker-configuration-file.md [hunk 5]
```diff
@@ -6,59 +6,35 @@ category: reference
...
+| `worker-addr` | Specifies the address of DM-worker which provides services. You can omit the IP address and specify the port number only, such as ":8262". |
+| `advertise-addr` | Specifies the address that DM-worker advertises to the outside world. |
```
Comment (TomShawn) [inline en/dm-worker-configuration-file.md:39]:
```suggestion
| `advertise-addr` | Specifies the address that DM-worker is advertised externally. |
```

**Comment by TomShawn** [comment]
@toutdesuite Please add the alias `['/docs/tidb-data-migration/dev/dm-worker-configuration-file-full/']` in the metadata section of `dm-worker-configuration-file.md`.

**Comment by toutdesuite** [comment]
@TomShawn Alias is added. PTAL again~

**Pull request merged**